src/actions/index.js

```diff
@@ -117,13 +119,21 @@ export function signOut() {
   }
 }
 
-export function fetchProjects(parms) {
+export function fetchProjects() {
   return dispatch => {
     dispatch(setError(''))
-    dispatch(setIsFetching(false))
+    var callFetchProjects = tag => dispatch(fetchProjectsByTag(tag.value))
+    forEach(callFetchProjects, filter(propEq('display', true), GLOBALS.DISCIPLINES))
+  }
+}
+
+
+export function fetchProjectsByTag(tag) {
+  const parms = {id: MOBILE_PROJECTS, cards: true, tags: tag, sort: 'display_name'}
```

Comment:
Yeah, I think they actually worked as a random list of projects at one point, but I think when we added the 'tags' query they stopped working altogether :-(  Right now I have the Panoptes client hardcoded to production in the /node_modules/, which is less than desirable, as I have to redo it everytime npm install is run 👎